docs/sources/mimir/release-notes/v2.15.md

```diff
@@ -0,0 +1,128 @@
+- `-query-scheduler.prioritize-query-components`, which is always enabled now
+- `-ingest-storage.kafka.ongoing-records-per-fetch` and `-ingest-storage.kafka.startup-records-per-fetch`, which have been removed in favour of `-ingest-storage.kafka.max-buffered-bytes`
+- `-api.get-request-for-ingester-shutdown-enabled`, a deprecated experimental flag which has been marked for removal in 2.15.
+
+## Experimental features
+
+Grafana Mimir 2.15 includes some features that are experimental and disabled by default.
+Use these features with caution and report any issues that you encounter:
+
+Mimir's experimental PromQL engine can now be enabled with `-querier.query-engine=mimir`. This new engine provides improved performance and reduced querier resource consumption; however, it supports only a subset of all PromQL features. It will fall back to Prometheus' engine for queries containing unsupported features.
+
+The query-frontend can now cache non-transient errors using the experimental flags `-query-frontend.cache-errors` and `-query-frontend.results-cache-ttl-for-errors`.
+
+The query-frontend and querier both support an experimental PromQL function, `double_exponential_smoothing`, which can be enabled by setting `-querier.promql-experimental-functions-enabled=true` and `-query-frontend.promql-experimental-functions-enabled=true`.
+
+The ingester can now support out-of-order native histogram ingestion via the flag `-ingester.ooo-native-histograms-ingestion-enabled`.
+
+The ingester can now build 24h blocks for out-of-order data which is >24h old, using the setting `-blocks-storage.tsdb.bigger-out-of-order-blocks-for-old-samples`.
+
+The ruler now supports caching the contents of rule groups via the setting `-ruler-storage.cache.rule-group-enabled`.
+
+The distributor now supports promotion of OTel resource attributes to labels via the setting `-distributor.promote-otel-resource-attributes`.
+
+## Bug fixes
+
+- Alerts: Fix autoscaling metrics joins in `MimirAutoscalerNotActive` when series churn.
+- Alerts: Exclude failed cache "add" operations from alerting since failures are expected in normal operation.
+- Alerts: Exclude read-only replicas from `IngesterInstanceHasNoTenants` alert.
+- Alerts: Use resident set memory for the `EtcdAllocatingTooMuchMemory` alert so that ephemeral file cache memory doesn't cause the alert to misfire.
+- Dashboards: Fix autoscaling metrics joins when series churn.
+- Distributor: Fix pooling buffer reuse logic when `-distributor.max-request-pool-buffer-size` is set.
+- Ingester: Fix issue where active series requests error when encountering a stale posting.
+- Ingester: Fix race condition in per-tenant TSDB creation.
+- Ingester: Fix race condition in exemplar adding.
+- Ingester: Fix race condition in native histogram appending.
+- Ingester: Fix bug in concurrent fetching where a failure to list topics on startup would cause to use an invalid topic ID (0x00000000000000000000000000000000).
```

Comment:
```suggestion
- Ingester: Fix bug in concurrent fetching where a failure to list topics on startup would cause an invalid topic ID (0x00000000000000000000000000000000).
```